message_ix/model/MESSAGE/model_core.gms

```diff
@@ -222,6 +222,8 @@ Set
 Positive variables
     SLACK_COMMODITY_BALANCE_UP(node,commodity,level,year_all,time) slack variable for commodity balance (upwards)
     SLACK_COMMODITY_BALANCE_LO(node,commodity,level,year_all,time) slack variable for commodity balance (downwards)
+    SLACK_COMMODITY_BALANCE_FULL_UP(node,commodity,level,year_all,time) slack variable for full commodity balance (upwards)
```

Comment:
Here it gets tricky since there is already `SLACK_COMMODITY_BALANCE_UP`. Perhaps `SLACK_COMMODITY_BALANCE_LT_UP`? @danielhuppmann thoughts?

Reply:
there is no need to duplicate the slack equations, they can be used for both the upper and lower part of the `COMMODITY_BALANCE` equation.